controls/postgres_spec.rb

```diff
@@ -66,10 +66,59 @@
   impact 1.0
   title 'Postgresql should be running'
   desc 'Postgresql should be running.'
-  describe service(postgres.service) do
-    it { should be_installed }
-    it { should be_running }
-    it { should be_enabled }
+  case os[:name]
+  when 'ubuntu'
+    case os[:release]
+    when '12.04'
+      describe command('/etc/init.d/postgresql status') do
+        its('stdout') { should include 'online' }
+      end
+      # describe upstart_service(postgres.service) do
```

Comment:
lets remove those comments and keep the original test commented out, because that is where we want to go back to. InSpec should handle that properly.